src/gs-category-page.c

```diff
@@ -214,6 +216,23 @@ gs_category_page_reload (GsPage *page)
 		gtk_widget_set_visible (self->infobar_category_shell_extensions, FALSE);
 	}
 
+
+	if (g_strcmp0 (gs_category_get_id (self->category), "usb") == 0) {
+		// FIXME: cut
```

Comment:
Use a C-style comment. Also, what does this mean?

Reply:
I leave myself a number of FIXMEs and TODOs while developing and always check for those strings before merging (and usually before sharing, hence why I didn't bother using C-style comments for these, generally don't bother wrapping them well, etc.).

At any rate, they're removed for the feature-complete version of the branch I'm about to push.